selinux/ceph.te

```diff
@@ -103,6 +103,7 @@ fstools_exec(ceph_t)
 nis_use_ypbind_uncond(ceph_t)
 storage_raw_rw_fixed_disk(ceph_t)
 files_manage_generic_locks(ceph_t)
+libs_exec_ldconfig(ceph_t)
```

Comment:
@b-ranto we run ldconfig before/after installing lib{rados,rbd,cephfs,rgw}, for instance, https://github.com/ceph/ceph/blob/master/ceph.spec.in#L1574 . but i think it is the rpm who is running `ldconfig`, not ceph. am i right? or i am looking at the run place?

Reply:
Yes, we do run ldconfig in the rpm scripts but that wouldn't cause this issue. The commands there are not run with the ceph_t context. The denial in the tracker issue reads

``{ execute } for  pid=16549 comm="sh" name="ldconfig" dev="sda1" ino=5201 scontext=system_u:system_r:ceph_t:s0 tcontext=system_u:object_r:ldconfig_exec_t:s0 tclass=file``

which means that a shell script with ceph_t context (which should imply it was run from within a ceph daemon) is trying to execute ldconfig but it is forbidden by SELinux to do so.

I poked around the ceph sources trying to find anything that would do this but I didn't find anything either (although there were some references to ldconfig in some of the libraries that we bundle). I was hoping somebody else might know more about this. In any case, allowing ceph to run ldconfig did not seem like a big security threat to me so I went ahead and created this PR.

Also, the timing for the denial does not match any rpm scripts that could be run at that point.

Now that I think about it, this might be the case of a test script running with ceph_t context trying to run ldconfig. It would even make more sense since the ldconfig denials are the only denials that reference dev as sda1, not dm-1.